content/en/docs/tasks/security/authentication/claim-to-header/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Copy Jwt Claim to HTTP Header
+description: Shows how users can copy their jwt claims to http headers.
+weight: 30
+keywords: [security,authentication,jwt,claim]
+aliases:
+    - /docs/tasks/security/istio-auth.html
+    - /docs/tasks/security/authn-policy/
+owner: istio/wg-security-maintainers
+test: yes
+status: Experimental
+---
+This task shows you how to copy valid JWT claims to http headers after JWT authentication is successfully completed via Istio request authentication policy.
+
+Note: Claims of type string, boolean, integer are supported. Array type claims are not supported as of now.
+
+## Before you begin
+
+Before you begin this task, do the following:
+
+* Complete the [Istio end user authentication task](/docs/tasks/security/authentication/authn-policy/#end-user-authentication).
```

Comment:
This doesn't seem like a necessary step in order to complete this task. Do you really mean something like this:
```suggestion
* Familiarize yourself with [Istio end user authentication](/docs/tasks/security/authentication/authn-policy/#end-user-authentication) support.
```